ansible/files/bootstrap/telemetry.sh

```diff
@@ -62,6 +26,14 @@ while [ $# -gt 0 ]; do
             shift
             TIMEOUT=$1
             ;;
+        --email)
+            shift
+            EMAIL=$1
+            ;;
+        --efs-dns)
+            shift
+            EFS_DNS=$1
```

Comment:
I think there should be a different EFS drive per environment to avoid conflicts.